apply/apply_test.go

```
@@ -218,9 +219,9 @@ func TestCreateFileNonExistentDirectory(t *testing.T) {
 }
 
 func TestApplySmokeTest(t *testing.T) {
-	a := assert.New(t)
+	r := require.New(t)
```

require fails fast, while assert keeps going